libqtile/layout/floating.py

```diff
@@ -231,12 +249,6 @@ def configure(self, client, screen_rect):
             client.paint_borders(bc, bw)
             client.cmd_bring_to_front()
 
-        # similar to above but the X11 version, the client may have already
-        # placed itself. let's respect that
-        elif client.has_user_set_position():
-            client.paint_borders(bc, bw)
-            client.cmd_bring_to_front()
-
         # ok, it's not java and the window itself didn't position it, but users
```

Comment:
This comment is now wrong, IIUC

Reply:
You are correct, thanks for spotting that. I changed the comment, too.